site/operational-guide/uniform-dynamic-cluster.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Uniform Dynamic Clusters
+menu_order: 50
+---
+
+A uniform dynamic cluster has the following characteristics:
+
+* Recovers automatically after reboots and partitions.
+* All peers have identical configuration.
+* Initial cluster peers can make progress from the outset even if
+  bootstrapping occurs under conditions of partition.
+* Once an initial cluster has been bootstrapped, arbitrary numbers of
+  new peers can be added in parallel without coordination. This makes
+  it ideally suited for use with convergent provisioning tools that
+  operate across multiple hosts in an asynchronous fashion.
+
+## Bootstrap
+
+On each initial peer, at boot, via
```

Comment:
Should we mention that there needs to be at least 3 initial peers for the instructions to be followed accurately?